xmppserver/src/main/java/org/jivesoftware/openfire/group/DefaultGroupProvider.java

```diff
@@ -369,7 +369,7 @@ public void addMember(String groupName, JID user, boolean administrator) throws
             con = DbConnectionManager.getConnection();
             pstmt = con.prepareStatement(ADD_USER);
             pstmt.setString(1, groupName);
-            pstmt.setString(2, server.isLocal(user) ? user.getNode() : user.toString());
+            pstmt.setString(2, server.isLocal(user) ? user.getNode() : user.asBareJID().toString());
```

Comment:
Although this makes sense to me, does this break existing behavior? Is there a chance that someone relies on this functionality?

Reply:
Nah.